src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/ActivatorUtilitiesTests.cs

```diff
@@ -391,15 +391,15 @@ public AbstractFoo()
             }
         }
 
-        private class Bar
+        private sealed class Bar
```

Comment:
This directory looks like a test project - could sealing these types have subtle behavioral differences the test project didn't intend?